### ISSUE TITLE
Blazor Wasm Not Working When Published to Azure

### ISSUE THREAD

**Issue by Anzel13** [dotnet/aspnetcore#29770]
When I create a brand new Blazor App using Visual Studio 2019 and deploy it to my Azure App service the "Fetch Data" / "Weather Forecast" returns an error.  When I look at the development tools I can see that the WeatherForecast end point is returning HTML whereas on my local machine it returns JSON.

This is a brand new / vanilla Blazor Web Assembly app this is hosted with a Blazor Server.  I'm not changing any of the code and I'm not modifying the configuration whatsoever.  Why am I getting JSON when I run it locally and then HTML when I deploy it to an Azure App Service?  Please help.

**Comment by Anzel13** [comment]
When looking at the developer tools (console) in Chrome I could see that there was an issue with serializing/deserializing  the Weather Forecast object.  Just as an experiment I installed the Newtonsoft.Json NuGet package and everything works perfectly.  I'm not sure why this is the case because again, I'm not changing any of the code.  Not one line.  Not even after installing Newtonsoft.

**STEPS TO REPRODUCE**
1.) Create a basic Blazor (Core Hosted) Web Assembly application.
2.) Test the "Fetch Data" locally
3.) Deploy the unmodified project to an Azure App Service
4.) Observe the errors when trying to Fetch Data
5.) Install the Newtonsoft.Json NuGet package for both the client and server project.
6.) Re-Deploy to the Azure App Service.
7.) Observe that "Fetch Data" now works perfectly.

I still have no idea why a.) the default project works locally but throws an error when deployed to Auzre and b.) why simply installing Newtonsoft corrects the problem.  I simply installed the package without changing the code at all.
